src/queries/types.js

```diff
@@ -13,3 +15,17 @@ export const QUERY_TYPES = {
   TOUR: 'tour',
   TOURS: 'tours'
 };
+
+/**
+ * Parse a query type from json string if one matches.
+ * @argument {string} input
```

Comment:
we use `@param` instead of its synonym `@argument`. we should keep it uniform.

Reply:
done in 5c4968a26c167de3f0343c155960812f53460942